src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -333,7 +332,7 @@ export const richNameValidationSchema = (t: TFunction, usedNames: string[], orig
   const nameValidationMessagesList = nameValidationMessages(t);
   return Yup.string()
     .min(1, nameValidationMessagesList.INVALID_LENGTH)
-    .max(253, nameValidationMessagesList.INVALID_LENGTH)
+    .max(63, nameValidationMessagesList.INVALID_LENGTH)
```

Comment:
Similar as with `nameValidationMessages` function, can we add a dedicated function for hostname `richHostnameValidationSchema`?